contracts/ResilientOracle.sol

```diff
@@ -115,7 +115,6 @@ contract ResilientOracle is PausableUpgradeable, AccessControlledV8, ResilientOr
      * @custom:error Throws a length error if the length of the token configs array is 0
      */
     function setTokenConfigs(TokenConfig[] memory tokenConfigs_) external {
-        _checkAccessAllowed("setTokenConfigs(TokenConfig[])");
```

Comment:
We should remove the granted permission also from the VIP: https://github.com/VenusProtocol/vips/pull/4